### ISSUE TITLE
Add a mix task which pulls blocks from ropsten via ex_wire and verifies them with `Blocktree.verify_and_add_block`

### ISSUE THREAD

**Issue by masonforest** [mana-ethereum/mana#36]
This will be the first step toward pulling together all of the pieces into a functioning client. This can be done in parallel with #3 and merged into the CLI once both are complete.

**Comment by hayesgm** [comment]
Closing in lieu of #550
